src/Tribe/Views/V2/Widgets/Widget_List.php

```diff
@@ -160,4 +162,25 @@ public function get_limit_options() {
 
 		return $options;
 	}
+
+	/**
```

Comment:
* :no_entry_sign: Doc comment for parameter "$arguments" missing
* :no_entry_sign: Doc comment for parameter "$context" missing

<sub>hash: 978817e56491e712d5623fa01f4dcf7e</sub>